components/VM_Arm/plat_include/tx2/plat/vmlinux.h

```diff
@@ -13,7 +13,7 @@
 #define GIC_LIC_INTID_BASE      (32)
 #define TX2_IRQ_PPI_VTIMER      (27)
 
-enum IRQConstants {
+enum platform_interrupt_t {
```

Comment:
This probably actually wants to be a typedef at the front, so that platform_interrupt_t can be used without the enum keyword. (https://docs.sel4.systems/processes/style-guide.html#type-conventions)

Reply:
Yes, that makes sense. Will fix.

Reply:
Done